### PR TITLE
Find Data - Load data from different sources #4a9p42 #4a7q6p

### PR DESCRIPTION
**Pull request by cameronbaney**
# Description

The purpose of this PR is to create a system to load data from different sources. We have two sources for the Find Data page: Contentful and the Blackfynn API.

In addition, I created the component to render Datasets.

**Note:** Filters do not work at the moment and will be in a separate PR when [the endpoint](https://app.clickup.com/t/4a7n77) is available
## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?
- Go to the Find Data page
- Datasets should be loaded (this is from the Blackfynn API)
- Use the pagination, and datasets should be loading
- Search for something like "Rat"
- Filtered datasets should be shown
- Go to the Projects tab and Projects should be loaded (this is from Contentful)
- Go to the Datasets tab and Datasets should be loaded

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by cameronbaney** [comment]
Waiting on #20 to be merged, as I needed some of that functionality to continue with this work.

File: pages/data/index.vue [hunk 1]
```diff
@@ -271,18 +282,59 @@ export default {
...
+    /**
+     * Get Search results
+     * This is using fetch from the Blackfynn API
+     */
+    fetchFromBlackfynn: function() {
+      this.isLoadingSearch = true
+
+      const searchType = pathOr('', ['query', 'type'], this.$route)
+      const query = pathOr('', ['query', 'q'], this.$route)
+      const url = `${process.env.discover_api_host}/search/${searchType}s?offset=${this.searchData.skip}&limit=${this.searchData.limit}&organization=SPARC%20Consortium&query=${query}`
```
Comment (mahamzayed) [inline pages/data/index.vue]: Can we make this a computed property instead?

Comment (cameronbaney) [reply pages/data/index.vue]: I kept it not a computed value because we need to pass some more conditional logic in to it, which you'll see in #22

Comment (mahamzayed) [reply pages/data/index.vue]: Sounds good, thanks for the clarification!

**Pull request merged**